LICENSE

```diff
@@ -1,20 +1,28 @@
-The MIT License (MIT)
+Copyright (c) 2016 Richard Feldman and Max Goldstein
+All rights reserved.
 
-Copyright (c) 2014 Alex Neslusan
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of
-this software and associated documentation files (the "Software"), to deal in
-the Software without restriction, including without limitation the rights to
-use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
-the Software, and to permit persons to whom the Software is furnished to do so,
-subject to the following conditions:
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+* Neither the name of elm-stylesheets nor the names of its
```

Comment:
Thanks! Fixed in https://github.com/elm-community/elm-test/pull/24/commits/f72463fc647bf277c05fa38786c6c8a59dc2a28b